src/test/java/test/reports/EmailableReporterTest.java

```diff
@@ -19,38 +19,38 @@
 public class EmailableReporterTest extends SimpleBaseTest {
     private SecurityManager manager;
 
-    @BeforeClass (alwaysRun = true)
+    @BeforeClass (alwaysRun=true)
```

Comment:
@cbeust - Ah ok! That was IntelliJ playing games when I did a format. Let me fix that.